lib/scripts/utils/languages.js

```diff
@@ -0,0 +1,103 @@
+export var bingLanguages = {
```

Comment:
Is it possible to fetch the list of languages supported by Yandex and Bing dynamically? Would it make sense to do it? Or not?